js/authentication.md

```diff
@@ -86,6 +86,55 @@ A configuration file called `aws-exports.js` will be copied to your configured s
 
 **NOTE**: If your Authentication resources were created with Amplify CLI version 1.6.4 and below, you will need to manually update your project to avoid Node.js runtime issues with AWS Lambda. [Read more]({%if jekyll.environment == 'production'%}{{site.amplify.docs_baseurl}}{%endif%}/cli/lambda-node-version-update)
 
+#### Configuring Cognito Lambda Triggers
+
+The CLI allows you to configure [Lambda Triggers](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools-working-with-aws-lambda-triggers.html) for your AWS Cognito User Pool.  These can enable you to add custom functionality to your registration and authentication flows.
+
+There are two ways to setup Lambda Triggers for your Cognito User Pool.
+
+1. In the default Auth CLI workflow, you will be presented with a list of prepared Lambda Trigger templates if you opt to configure advanced settings:
```

Comment:
you will be presented with a list of Lambda Trigger templates

Reply:
I took out 'prepared', but left 'if you opt to configure advanced settings:'.